pkgs/applications/gis/grass/default.nix

```diff
@@ -6,24 +6,33 @@
 
 stdenv.mkDerivation rec {
   pname = "grass";
-  version = "8.2.0";
+  version = "8.2.1";
 
   src = with lib; fetchFromGitHub {
     owner = "OSGeo";
     repo = "grass";
     rev = version;
-    sha256 = "sha256-VK9FCqIwHGmeJe5lk12lpAGcsC1aPRBiI+XjACXjDd4=";
+    sha256 = "sha256:02fxcpdwkjis0i5mqks1by4kwazi1947flmhfd0yz2xxgdvx0wsk";
```

Comment:
I suggest changing `src.sha256` attribute to `src.hash` which I believe is the newer standard.

Also, the hash value I've got by using `lib.fakeHash` trick doesn't match hash in this PR:
```
    hash = "sha256-U3PQd3u9i+9Bc7BSd0gK8Ss+iV9BT1xLBDrKydtl3Qk=";
```

Reply:
It's a different encoding of the same hash. I use `nix (Nix) 2.3.16` on the machine I did my build on; my machines with newer Nix also generate the other form of hash.

Converting:

```sh
nix --extra-experimental-features nix-command hash to-base64 'sha256:02fxcpdwkjis0i5mqks1by4kwazi1947flmhfd0yz2xxgdvx0wsk'
```

Output:
```
U3PQd3u9i+9Bc7BSd0gK8Ss+iV9BT1xLBDrKydtl3Qk=
```

I've switched it and also use the new `hash` now.